sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -965,9 +965,27 @@ class JDBCSuite extends SparkFunSuite
           |CREATE TEMPORARY VIEW people_view
           |USING org.apache.spark.sql.jdbc
           |OPTIONS (uRl '$url', DbTaBlE 'TEST.PEOPLE', User 'testUser', PassWord 'testPass')
-        """.stripMargin.replaceAll("\n", " "))
+        """.stripMargin)
 
       assert(sql("select * from people_view").count() == 3)
     }
   }
+
+  test("ALTER TABLE ADD COLUMNS") {
```

Comment:
are we going to test all the unsupported data sources? that's a lot, and unnecessary. I think the text format test is enough, let's remove others.